src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceDefault.java

```diff
@@ -1071,4 +1071,10 @@ private void n() {
         New6.field = (int)a;
     }
 
+    class SwitchStatement {
```

Comment:
Please do not extend these input files, make a new one with exact config from issue description. We are trying to reduce the size of our input files, let's not add to our tech debt with this update.

Reply:
Done.

Reply:
>partially https://github.com/checkstyle/checkstyle/issues/11163

No. This PR should consist of only:
1. code change in check
2. new unit test
3. new input with config and example from issue report (please expand a bit to include switch expression, etc.)
4. removal of pitest suppression

Reply:
Okay, didn't know you are not okay with cleaning up technical debt along with solving issues around. Will keep this in mind for the future. Restored the commit and added more test cases. Done.